src/packages/ganache/README.md

```diff
@@ -205,10 +194,10 @@ const provider = new ethers.providers.Web3Provider(ganache.provider());
 You can also use Ganache in the browser by adding the following script to your HTML:
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/ganache@{VERSION}/dist/web/ganache.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/ganache@latest/dist/web/ganache.min.js"></script>
 ```
 
-NOTE: the `{VERSION}` in the above path needs to be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions).
+NOTE: the `latest` in the above path can be replaced with a version number or tag that is listed in [npm](https://www.npmjs.com/package/ganache?activeTab=versions).
```

Comment:
I'd prefer something like `@{VERSION}` because we _don't_ want the link to work as is. For stability and security should reference a pinned version.